administrator-manual/en/ui/Ejabberd.rst

```diff
@@ -0,0 +1,22 @@
+.. _ChatUi-section:
+
+====
+Chat
+====
+
+The :index:`chat` service uses the standard protocol :index:`Jabber`/:index:`XMPP`. See also
+:ref:`chat-section`.
+
+
+Administration web Interface
+    The Jaber server comes with a web administrative interface for members of the jabberadmins group.
+
+Federation (S2S)
+    XMPP allows for servers communicating seamlessly with each other, forming a global 'federated' IM network.
+
+File transfert maximum speed
```

Comment:
Multiple occurrences of :
transfert -> transfer

Reply:
bloody french, too close of english.....Normal we ruled the Britain so much years

Reply:
Aahahaha

Reply:
Aahahahahhahahahahaah